setup.py

```diff
@@ -132,7 +132,6 @@ def get_extras_require() -> Dict[str, List[str]]:
             "fakeredis<=1.7.1; python_version<'3.8'",
             "fakeredis ; python_version>='3.8'",
             "pytest",
-            "pytest-cov",
```

Comment:
https://github.com/codecov/codecov-python/blob/12c0dddd6350a1d511f22d363e89d9019b149164/setup.py#L45

[memo] `coverage` is installed as the dependency of `codecov`. Although it might beyond this PR's context, it might be safer to add `coverage` in `test` extras.